### DISCUSSION TITLE
Carla not Rendering properly in UBUNTU 20.04

### DISCUSSION THREAD

**Discussion by newton2149** [carla-simulator/carla#5116] [General]
![Screenshot from 2022-01-29 14-50-44](https://user-images.githubusercontent.com/64192353/151655566-9e65e1d0-5b4d-484f-b037-d7145b36e1a3.png)
Carla Not rendering properly
Why am I getting this error ?
System Specs
Intel i7 11th Gen
NVIDIA 3050 Ti
16gb Ram
OS Ubuntu 20.04

**Comment by lhmwtum** [comment]
This seems to be related to a graphics driver issue, see here: #4527

**Comment by xiaoqingnianbudadun** [comment]
try to reinstall the nvidia driver follow this article
https://zhuanlan.zhihu.com/p/447499750
